### DISCUSSION TITLE
Server Side Prefix Can be Bypassed by Using the Prefix That You Made The Default

### DISCUSSION THREAD

**Discussion by Koderew** [jagrosh/MusicBot#1483] [Ideas]
I was playing with the bot when I accidentally used the default prefix I set for the bot instead of the server-side one. (There's a different prefix since there are two of these music bots in the server.) I find this annoying, as users who may not know that there's a different prefix may accidentally make both bots play music. It would be really nice if the bot had a server-side prefix and would only use that prefix for that server, and wouldn't use the default prefix the bot has just for that server.

**Comment by jagrosh** [comment]
The server prefix is not overriding the default prefix, but rather is an additional choice. This won't be changed. If you leave the default prefix as @mention it is extremely unlikely that it will collide with other bots.
